platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/ConsensusImpl.java

```diff
@@ -141,6 +142,8 @@
  */
 public class ConsensusImpl extends ThreadSafeConsensusInfo implements Consensus {
 
+    public static final String COIN_ROUND_MARKER_FILE = "consensus-coin-round";
+    public static final String NO_SUPER_MAJORITY_MARKER_FILE = "consensus-no-super-majority";
```

Comment:
Do we expect for whatever utility is watching for these marker files to do things differently depending on which marker file is found? If "no", then would it make more sense to instead use the same marker file type for each mode of failure?

Reply:
I think it should always be the same file, but maybe the contents can change depending on the issue

Reply:
marker files typically do not have content.   It is simply presence / absence of a named file.   JRS will detect that the directory is non-empty, but the human triaging the failure will want to know the cause.  Without differentiation of file name, we will not know the root cause of the data dump.

Reply:
I'm ok resolving this. Mostly was trying to understand better how we intended to use markers.